src/server/scripts/Commands/cs_cheat.cpp

```diff
@@ -214,15 +210,17 @@ class cheat_commandscript : public CommandScript
 
     static bool HandleTaxiCheatCommand(ChatHandler* handler, const char* args)
     {
+        std::string argstr = (char*)args;
+        
         if (!*args)
         {
+            // Use Function player->isTaxiCheater() (bool)
+            // argstr = (handler->GetSession()->GetPlayer()->GetCommandStatus(CHEAT_TAXICHEAT)) ? "off" : "on";
             handler->SendSysMessage(LANG_USE_BOL);
             handler->SetSentErrorMessage(true);
             return false;
         }
 
-        std::string argstr = (char*)args;
```

Comment:
its bad to remove it since later it is compared: `argstr == "off"` and `char*` doesnt work like that.